build-tools/packages/build-cli/src/commands/generate/compatLayerGeneration.ts

```diff
@@ -41,7 +42,7 @@ export default class UpdateGenerationLayerCommand extends PackageCommand<
 			default: `layerGenerationState.ts`,
 		}),
 		minimumCompatWindowMonths: Flags.integer({
-			description: `The minimum compatibility window in months that is supported across all Fluid layers.`,
+			description: `The minimum compatibility window in months that is supported across all Fluid layers. Must be greater than 0`,
 			default: 3,
```

Comment:
You can use the `min` property to specify the min and the infra will enforce it for you.

Reply:
You can also specify a max if useful.